### PR TITLE
Cleanup and refactor

### PR DESCRIPTION
**Pull request by ggdaltoso**
As discussed in #205, we need to properly handle some issues with `CSS` files, from  `@react95/icons` package and also with `PNGs` and `SVGs`.

I ended up transforming `PNGs` and `SVGs` as `base64` and, regardless of what I thought, I had very nice results.
image

For the final packages, the built ones, we also have de CJS and ESM exports and a folder just for the types (@types)

Since this PR contains a breaking change, I changed the build process to create an alpha release when the branch `alpha` gets changed by a PR. So, I'm adding this automatization as well.

### CODE REVIEW

**Comment by allyssonsantos** [comment]
@ggdaltoso What a nice work, thanks!

I think that you must add the [`sideEffects`](https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free) flag inside the `package.json` files, or in the `prepublish` script to enable `tree-shaking` properly.

**Comment by ggdaltoso** [comment]
Good point, @allyssonsantos 

I'll add `sideEffects` for the font files

**Review by allyssonsantos — APPROVED** [review]

File: packages/core/README.md [hunk 1]
```diff
@@ -44,3 +44,25 @@ yarn add @types/styled-components @types/xstyled__styled-components
 ```
 
 You can find a list of all available components at our [Storybook](https://react95.github.io/React95/)
+
+### SSR
```
Comment (tcK1) [inline packages/core/README.md:48]: Not necessarily SSR, right? This is a Next.js example.

Comment (ggdaltoso) [reply packages/core/README.md]: Done!

**Pull request merged**